### PR TITLE
[ISSUE #2284]Simplify the code

### PR DESCRIPTION
**Pull request by weihubeats**
Fixes https://github.com/apache/incubator-eventmesh/issues/2284

### Motivation

*Explain the content here.*
*Explain why you want to make the changes and what problem you're trying to solve.*

### Modifications

*Describe the modifications you've done.*

### Documentation

- Does this pull request introduce a new feature? (yes / no)
- If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
- If a feature is not applicable for documentation, explain why?
- If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

**Comment by xwm1992** [comment]
@weihubeats why you add the `Assert` class, junit has already done it.

**Comment by weihubeats** [comment]
Assert is only used for testing, the added Assert is used for null checking in the program, or we can rename the Assert to AssertUtils

**Comment by xwm1992** [comment]
> Assert is only used for testing, the added Assert is used for null checking in the program, or we can rename the Assert to AssertUtils

Ok, maybe rename to `AssertUtils` is better.

**Comment by weihubeats** [comment]
ok

File: eventmesh-common/src/test/java/org/apache/eventmesh/common/utils/AssertTest.java [hunk 1]
```diff
@@ -20,17 +20,17 @@
 import org.junit.Test;
 
 /**
- * test {@link Assert}
+ * test {@link AssertUtils}
  */
 public class AssertTest {
```
Comment (jonyangx) [inline eventmesh-common/src/test/java/org/apache/eventmesh/common/utils/AssertTest.java:25]: suggest rename AssertUtilsTest.

File: eventmesh-common/src/test/java/org/apache/eventmesh/common/utils/AssertTest.java [hunk 2]
```diff
@@ -20,17 +20,17 @@
 import org.junit.Test;
 
 /**
- * test {@link Assert}
+ * test {@link AssertUtils}
```
Comment (jonyangx) [inline eventmesh-common/src/test/java/org/apache/eventmesh/common/utils/AssertTest.java:23]: suggest rename AssertUtilsTest.

**Review by xwm1992 — APPROVED** [review]

**Pull request merged**